script/script.js

```diff
@@ -0,0 +1,71 @@
+const titleEl = document.getElementById('titleEl');
+const authorEl = document.getElementById('authorEl');
+const form = document.getElementById('my-form');
+const bookList = document.querySelector('.book-list');
+
+/* new code with class */
+// let dataLink = Math.random();
```

Comment:
- [ ] [OPTIONAL] please avoid committing commented out code to your repo as it makes your codebase look unorganized and harder to read.